app/src/main/java/io/github/wiiznokes/gitnote/ui/screen/app/grid/GridScreen.kt

```diff
@@ -181,21 +188,25 @@ private fun GridView(
     onEditClick: (Note, EditType) -> Unit,
     selectedNotes: List<Note>,
     padding: PaddingValues,
+    noteViewType: NoteViewType,
 ) {
     val gridNotes = vm.gridNotes.collectAsLazyPagingItems()
 
     val gridState = rememberLazyStaggeredGridState()
+    val listState = rememberLazyListState()
```

Comment:
honestly, i don't think it's a good idea to maitain 2 states at the same time like this. Can you initialize them in their code path ? That means duplicate LaunchedEffect